riotwatcher/_apis/VersionApi.py

```diff
@@ -0,0 +1,24 @@
+from . import NamedEndpoint
+
+
+class VersionApi(NamedEndpoint):
```

Comment:
Why is this a different API instead of part of DDragon?

Not saying is has to be a part of DDragon, I just want to know the reasoning.

Reply:
The reason why is because the URLs for the version and for the actual file a a bit different. The version take only the region as a path and the static has version, locale and finally the url_extension. I will have to call both request_version and request_static from DataDragonApi, but I can see that it would be maybe more confusing for the use. I'll be happy to refactor it should you wish so.

Reply:
yeah i think since the Version API is only relevant to DDragon, I think it makes sense to include in there. I see why it was separate from an implementation standpoint, but i think to an end user, they dont really care that the URL is actually different, they care that version is relevant specifically to ddragon.